README.md

```diff
@@ -25,7 +25,10 @@ The [CXX-Qt Book](https://kdab.github.io/cxx-qt/book/getting-started/index.html)
 step-by-step and documents CXX-Qt's features for the latest release. The [examples folder](./examples) contains
 demonstrations of using threading, QQmlExtensionPlugin, and various other features.
 
-CXX-Qt is in early development and the API changes frequently. For the latest documentation, [install mdBook](https://rust-lang.github.io/mdBook/guide/installation.html)
+CXX-Qt is tested on CI on Linux, Windows, and macOS (all on x86). It should work on other platforms that Qt and
```

Comment:
"tested" on Windows is maybe a little strong ... it builds on Windows :-) the examples do run, however they segfault on exit IIRC.

Reply:
I just tested on Windows and they do not segfault on exit. However, some more instructions are needed to set all the environment variables Windows needs...

Reply:
I think this language is fine for now, but we do need to automate/document getting set up on Windows later.